spec/support/matchers.rb

```diff
@@ -0,0 +1 @@
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
Is this needed because `.to change { ... }` can not be changed with `.to_not change {}`?